core/src/com/ericc/the/game/mapGenerator/Tile.java

```diff
@@ -0,0 +1,30 @@
+package com.ericc.the.game.mapGenerator;
+
+public class Tile {
+    private boolean active;
+
+    Tile(boolean active) {
+        this.active = active;
```

Comment:
może lepiej passable niż active?

Reply:
pytanie czy chcemy zostawić 0 - 1, czy już migrowaqć z tego na coś bardziej zaawansowanego, typu 0 - not passable, 1 - corridor, 2 - room, 3 - doors etc.